puzzlesolver/puzzles/bishop.py

```diff
@@ -0,0 +1,290 @@
+from copy import deepcopy
+from . import ServerPuzzle
+from ..util import *
+from ..solvers import IndexSolver
+from ..puzzleplayer import PuzzlePlayer

```

Comment:
This dependency is not being used here and is more just a way to "play" the puzzle. That's why we don't need the PuzzlePlayer dependency.